databricks/sdk/dbutils.py

```diff
@@ -163,6 +163,29 @@ def listScopes(self) -> typing.List[SecretScope]:
         return [SecretScope(v.name) for v in self._api.list_scopes()]
 
 
+class _JobsUtil:
+    """Remote equivalent of jobs util"""
+
+    class _TaskValuesUtil:
+        """Remote equivalent of task values util"""
+
+        def get(self, taskKey: str, key: str, default: any = None, debugValue: any = None) -> None:
+            """
+            Returns `debugValue` if present, throws an error otherwise as this implementation is always run outside of a job run
+            """
+            if debugValue is None:
+                raise ValueError('No debug value set for task value, when outside of job run')
```

Comment:
Please throw the exact same error and message that DBR does:

```suggestion
                raise TypeError('Must pass debugValue when calling get outside of a job context. debugValue cannot be None.')
```

Based on:

![image](https://github.com/databricks/databricks-sdk-py/assets/102973110/f0156517-ff50-4baa-b7c5-54b73fcc3d4f)